german/Hilfeschrei/Vertrauen/insMaul/ins_maul

```diff
@@ -0,0 +1,2 @@
+Zum Glück haben Sie eine kleine Taschenlampe am Schlüsselbund und beginnen
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.